src/test/java/ca/uhn/fhir/jpa/starter/OperationCollectDataTest.java

```diff
@@ -168,7 +295,27 @@ private void postResource(String theUrl, String filePath)
 		}
 	}
 
-	private String readJsonFile(String path) throws IOException {
+	private CloseableHttpResponse postXmlResource(String theUrl, String filePath)
+      throws IOException{
+    HttpPost post = new HttpPost(theUrl);
+    String xml = readFile(filePath);
+
+    StringEntity entity = new StringEntity(xml);
+    post.setEntity(entity);
+    post.setHeader("Accept", "application/xml");
+    post.setHeader("Content-type", "application/xml");
```

Comment:
if   a`pplication/xml ` fails to return a json response , try  `application/fhir+xml`